src/juxt/site/alpha/init.clj

```diff
@@ -239,6 +236,75 @@
 (def base-uri-parser
   (p/complete
    (p/into {}
-    (p/sequence-group
-     (p/pattern-parser #"(?<scheme>https?)://" {:group {:juxt.reap.alpha.rfc7230/scheme "scheme"}})
-     host-parser))))
+           (p/sequence-group
+            (p/pattern-parser #"(?<scheme>https?)://" {:group {:juxt.reap.alpha.rfc7230/scheme "scheme"}})
+            host-parser))))
+
+(defn do-action [crux-node subject action & args]
+  (apply authz/do-action crux-node {::pass/subject subject} action args))
+
+(defn openid-provider-configuration-url
+  "Returns the URL of the OpenID Provider Configuration Information."
+  [issuer-id]
+  ;; See https://openid.net/specs/openid-connect-discovery-1_0.html#rfc.section.4.1
+  ;;
+  ;; "If the Issuer value contains a path component, any terminating / MUST be
+  ;; removed before appending /.well-known/openid-configuration."
+  ;;
+  ;; This uses a reluctant regex qualifier.
+  (str (second (re-matches #"(.*?)/?" issuer-id)) "/.well-known/openid-configuration"))
+
+(comment
+  (openid-provider-configuration-url "dev-14bkigf7.us.auth0.com"))
+
+(defn install-openid-provider! [crux-node issuer-id]
```

Comment:
Is this function needed, isn't this handled by the install script?

Reply:
It's mentioned [here](https://github.com/juxt/site/blob/actions/doc/next/OpenIdAuthentication.adoc#registering-the-openid-provider) in the setup info which is what I believe I used to setup mine but I'll keep looking to see if it's needed.